### PR TITLE
Update pestleAndMortar.plugin.kts

### PR DESCRIPTION
**Pull request by raynna**
## What has been done?
Fix so you can crush one item with pestle n mortal

### CODE REVIEW

File: game/plugins/src/main/kotlin/gg/rsmod/plugins/content/items/pestle_and_mortar/pestleAndMortar.plugin.kts
Comment (Tyluur) [inline game/plugins/src/main/kotlin/gg/rsmod/plugins/content/items/pestle_and_mortar/pestleAndMortar.plugin.kts:1]: File names should follow the naming convention used throughout the project.

File: game/plugins/src/main/kotlin/gg/rsmod/plugins/content/items/pestle_and_mortar/pestleAndMortar.plugin.kts [hunk 1]
```diff
@@ -11,7 +11,7 @@ sources.forEach { source ->
     on_item_on_item(Items.PESTLE_AND_MORTAR, source) {
         // If the player only has one of the source item, grind it immediately
         if (player.inventory.getItemCount(source) == 1) {
-            grindItem(player, source, 1)
+            grindItem(player, definitions[source]!!.result, 1)
```
Comment (Tyluur) [inline game/plugins/src/main/kotlin/gg/rsmod/plugins/content/items/pestle_and_mortar/pestleAndMortar.plugin.kts:14]: Is it necessary to use the not-null assertion operator (!!) in this case? 
Or is there a better way you could've written this?

https://kotlinlang.org/docs/null-safety.html#the--operator

**Review by Tyluur — CHANGES_REQUESTED** [review]

**Review by alycii — COMMENTED** [review]
updated file nime

File: game/plugins/src/main/kotlin/gg/rsmod/plugins/content/items/pestle_and_mortar/pestleAndMortar.plugin.kts
Comment (raynna) [reply game/plugins/src/main/kotlin/gg/rsmod/plugins/content/items/pestle_and_mortar/pestleAndMortar.plugin.kts:1]: I did not make the pestle and mortal, i just applied the fix

**Pull request merged**